ci-operator/templates/cluster-launch-e2e-azure.yaml

```diff
@@ -0,0 +1,197 @@
+kind: Template
```

Comment:
Add a comment that this is used for testing in openshift-azure. In its current form, we cannot reuse for launching clusters in origin tests.